conf/documentation.conf

```diff
@@ -1011,6 +1011,48 @@ description=<<EOT
 Password for the mysql database used by PacketFence.
 EOT
 
+[database.key_buffer_size]
+type=numeric
+description=<<EOT
+The key_buffer_size MySQL configuration attribute (in MB). Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
+[database.innodb_buffer_pool_size]
+type=numeric
+description=<<EOT
+The innodb_buffer_pool_size MySQL configuration attribute (in MB). Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
+[database.innodb_additional_mem_pool_size]
+type=numeric
+description=<<EOT
+The innodb_additional_mem_pool_size MySQL configuration attribute (in MB). Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
+[database.query_cache_size]
+type=numeric
+description=<<EOT
+The query_cache_size MySQL configuration attribute. Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
+[database.thread_concurrency]
+type=numeric
+description=<<EOT
+The thread_concurrency MySQL configuration attribute. Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
+[database.max_connections]
+type=numeric
+description=<<EOT
+The max_connections MySQL configuration attribute. Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
+[database.table_cache]
+type=numeric
+description=<<EOT
+The table_cache MySQL configuration attribute. Only change if you know what you are doing. Will only affect a locally running MySQL server.
+EOT
+
```

Comment:
We are missing (at least) two important values:
max_allowed_packet=16M
thread_cache_size=50